priv/js/nitro.js

```diff
@@ -15,13 +15,13 @@ function validateSources() { return true; }
 function querySourceRaw(Id) {
     var val, el = document.getElementById(Id);
     if (!el) {
-       val = qs('input[name='+Id+']:checked'); val = val ? val.value : "";
+       val = qs('input[id='+Id+']:checked'); val = val ? val.value : "";
```

Comment:
Looks like this shouldn't work:
If there is ```input[id='+Id+']``` than ```el = document.getElementById(Id)``` will not return undefined and this ```if(!el)``` will fail and we will never reach this statement.

Reply:
Given that, should the name attribute fallback to ID value when it is not explicitly set in element_radio, element_checkbox etc.? It might violate coding rule of explicit code though..

Reply:
The name attribute should be set if you want to make a group of radio-buttons. Otherwise use checkboxes with appropriate unique ids.

Reply:
When I did that, nitro would return only empty value. I will check tomorrow to make sure but it seems like current version may not be working.

Reply:
Yeah, example would be helpful!